torch/csrc/api/include/torch/nn/modules/embedding.h

```diff
@@ -40,7 +52,7 @@ class TORCH_API EmbeddingImpl : public torch::nn::Cloneable<EmbeddingImpl> {
   EmbeddingOptions options;
 
   /// The embedding table.
-  Tensor weight;
+  //Tensor weight;
```

Comment:
This is the internal weight attribute of `EmbeddingImpl` and is different from the `_weight` parameter that the user can pass in from somewhere else. We should likely keep it.